### PR TITLE
Move constraints from the Admin section to top of nav

### PR DESCRIPTION
**Pull request by davidoliverSP2**
Depends on https://github.com/neo-technology/neo4j-manual-modeling/pull/2152

### CODE REVIEW

File: cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+
+[[administration-constraints-syntax-list]]
+== Syntax for listing constraints
+
+List constraints in the database, either all or filtered on type.
+
+[source, cypher, role=noplay]
+----
+SHOW [ALL\|UNIQUE\|NODE [PROPERTY] EXIST[ENCE]\|RELATIONSHIP [PROPERTY] EXIST[ENCE]\|[PROPERTY] EXIST[ENCE]\|NODE KEY] CONSTRAINT[S]
```
Comment (Hunterness) [inline cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc]: While I remember it, I found a tiny error in this syntax line:
```suggestion
SHOW [ALL\|UNIQUE\|NODE [PROPERTY] EXIST[ENCE]\|REL[ATIONSHIP] [PROPERTY] EXIST[ENCE]\|[PROPERTY] EXIST[ENCE]\|NODE KEY] CONSTRAINT[S]
```

File: cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc [hunk 2]
```diff
@@ -0,0 +1,143 @@
+[[administration-constraints-syntax]]
```
Comment (Hunterness) [inline cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc:1]: should all of these links be updated to not include `administration`? (also goes for the links in the index and examples files)

File: cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+== Syntax for listing constraints
+
+List constraints in the database, either all or filtered on type.
+
+[source, cypher, role=noplay]
+----
+SHOW [ALL\|UNIQUE\|NODE [PROPERTY] EXIST[ENCE]\|RELATIONSHIP [PROPERTY] EXIST[ENCE]\|[PROPERTY] EXIST[ENCE]\|NODE KEY] CONSTRAINT[S]
+    [YIELD { * \| field[, ...] } [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
+    [WHERE expression]
+    [RETURN field[, ...] [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
```
Comment (Hunterness) [inline cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc]: If it is no longer in a table we don't need to escape the pipes, `\|`

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala [hunk 4]
```diff
@@ -113,7 +113,7 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
       text =
         """
           |One way of filtering the output from `SHOW CONSTRAINTS` by constraint type is the use of type keywords,
-          |listed in the <<administration-constraints-syntax, syntax table>>.
+          |listed in the <<administration-constraints-syntax-list, _Syntax for listing constraints_ example>>.
```
Comment (Hunterness) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala]: not sure `example` is the correct word here

File: cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc (hunk 2 above)
Comment (davidoliverSP2) [reply cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc:1]: Yes, I think so, but I think this should be handled in another PR, post publication

**Pull request merged**